<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Manual Reduction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="5">
         <widget class="QCheckBox" name="checkBox_defaultCalibrationFile">
          <property name="text">
           <string>Default</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mask File</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEdit_vanRunNumber">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vanadium run number (integer) for detector pixels' efficiecy correction.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QCheckBox" name="checkBox_defaultMaskFile">
          <property name="text">
           <string>Default</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_browseCalibrationFile">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="lineEdit_outputDirectory">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Run Number</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="pushButton_browseOutputDirectory">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_browseMaskFile">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_calibrationFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Calibration File</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_browseNeXus">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_maskFile">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Output DIR</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QCheckBox" name="checkBox_defaultOutputDirectory">
          <property name="text">
           <string>Default</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Vanadium</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="pushButton_browseVanadium">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_runNumber">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run number or NeXus file name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButton_splitConvertSaveProject">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Split (sub run) / Convert (powder pattern) / Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_reduceRuns">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Simple_Reduction">
         <attribute name="title">
          <string>Simple Reduction</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4"/>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_subRunInfoTable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget_View">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Detector View</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_detectorView">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_detViewInfo">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Reduced Data</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_diffractionView">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Sub-Runs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_sub_runs"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_plotDetView">
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_nexus_file"/>
    <addaction name="actionLoad_Project_File"/>
    <addaction name="actionLoad_Image"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project_File"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Calibration</string>
    </property>
    <addaction name="actionEdit_Calibrations"/>
    <addaction name="actionFix_Calibrations"/>
   </widget>
   <widget class="QMenu" name="menuData_Reduction">
    <property name="title">
     <string>Data Reduction</string>
    </property>
    <addaction name="actionLoad_Instrument"/>
   </widget>
   <addaction name="menuQuit"/>
   <addaction name="menuEdit"/>
   <addaction name="menuData_Reduction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image ...</string>
   </property>
  </action>
  <action name="actionEdit_Calibrations">
   <property name="text">
    <string>Edit Calibrations</string>
   </property>
  </action>
  <action name="actionFix_Calibrations">
   <property name="text">
    <string>Fix Calibrations</string>
   </property>
  </action>
  <action name="actionLoad_Instrument">
   <property name="text">
    <string>Load Instrument</string>
   </property>
  </action>
  <action name="actionLoad_Project_File">
   <property name="text">
    <string>Load Project File ...</string>
   </property>
  </action>
  <action name="actionLoad_nexus_file">
   <property name="text">
    <string>Load NeXus File ...</string>
   </property>
  </action>
  <action name="actionSave_Project_File">
   <property name="text">
    <string>Save Project File ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
